modules/shortcodes/crowdsignal.php

```diff
@@ -1,610 +1,686 @@
-<?php
-
-// Keep compatibility with polldaddy-plugin
-if ( ! class_exists( 'CrowdsignalShortcode' ) && ! class_exists( 'PolldaddyShortcode' ) ) {
-
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
 /**
-* Class wrapper for Crowdsignal shortcodes
-*/
+ * Crowosignal (ne PollDaddy) shortcode.
```

Comment:
Tiny typo here should be "Crowdsignal".

Reply:
Fixed in 6c160d12b4d945eadc77f1691cb12eec8eb37ad8